scripts/GetPackageVersion.ps1

```diff
@@ -0,0 +1,4 @@
+$currentVersionString = (Get-Content  $(System.DefaultWorkingDirectory)/typingsBetaBuild/drop/package.json) -join "`n" | ConvertFrom-Json | Select -ExpandProperty "version"
```

Comment:
Does `$(System.DefaultWorkingDirectory)` work?

Variables work with inline PowerShell scripts, but I thought you need to use environment variables to access these in files.

Reply:
Changed this.